models/pardot/model.sql

```diff
@@ -81,3 +81,24 @@ create or replace view pardot_analysis.visitoractivity_events_meta as (
   select 18, '', 'reopened opportunity'
 
 );
+
+
+create or replace view {schema}.visitoractivity as (
+  --this table has a bunch of types that really should be event actions but are very poorly formulated.
+  --the custom logic in this view is an attempt to fix that.
+  --not all of the various type / type_name combinations have been accounted for yet; I still need to determine exactly what some of them mean.
+  select
+    -- event_stream interface
+    va.created_at       as "@timestamp",
```

Comment:
I don't have strong opinions here. It's kinda shitty because I do like the @ a lot as I've started to use it. Whatever you both think is the best approach.